src/app/containers/StoryPromo/MediaIndicator/index.test.jsx

```diff
@@ -146,6 +159,16 @@ describe('Story Promo Media Indicator', () => {
     />,
   );
 
+  shouldMatchSnapshot(
+    'should render correctly even without duration and cps type',
```

Comment:
Seems like this should be an assertion test rather than snapshot. Someone could easily just update the snapshots with checking it

Reply:
Yeah I'll update it